packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -317,7 +318,7 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ title, patientUuid, sh
               {showPrintButton && (
                 <Button
                   kind="ghost"
-                  renderIcon={(props) => <Printer size={16} {...props} />}
+                  renderIcon={(props) => <PrinterIcon {...props} size={16} />}
```

Comment:
Out of curiosity, is the design for iconography from the framework meant to inherit designs from the parent? for example in this case the Button stylings like color?

Reply:
They should inherit them, but I'm still untangling the mess of CSS to get things right. In this case, Carbon defines a selector like this:

```css
.cds--btn--ghost:not([disabled]) svg {
  fill: var(--cds-icon-primary, #161616);
}
```

But also setting `fill="currentColor"` on the icon, so it effectively ignores the CSS. Originally, I was doing something similar too, but it caused issues with getting the color right in some other cases. I'm still fiddling with getting this working in all cases...

Reply:
The `props` here are basically passing in some additional padding...